ibc-clients/cw-context/src/types/response.rs

```diff
@@ -1,12 +1,12 @@
 //! Contains the response types for the CosmWasm contract.
 use cosmwasm_schema::cw_serde;
-use ibc_client_wasm_types::Bytes;
```

Comment:
Ideally, we should remove the `ibc_client_wasm_types::Bytes` altogether. I tried to do that and noticed, our `Bytes` are being used at other places too.

PS. Probably, you are still working on the PR. In that case, ignore this comment.

Reply:
as talked and commented [here](https://github.com/cosmos/ibc-rs/issues/1271#issuecomment-2214573623) to be able to use 08-wasm types in no_std envs, we can't quite take a consistent approach at the moment.